src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      method="post"
+      action="http://google.pl"
+      data-qa="big"
+    >
+      <label
+        class="label"
+        for="search">
+        <img
+           src="images/Search.svg"
+           alt="search icon"
+           class="search-icon"
+           id="search"
+      >
+        <input
+          class="input"
+          type="search"
+          name="search"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+      >
```

Comment:
Fix indentation and add id so label is associated with this input
```suggestion
        <input
          id="search"
          class="input"
          type="search"
          name="search"
          placeholder="Try &quot;Los Angeles&quot;"
          data-qa="keypress"
        >
```